Resources/Prototypes/_Shitmed/Entities/Structures/Walls/walls.yml

```
@@ -55,13 +55,35 @@
+        node: girder
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalSlam
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
 
 - type: entity
   parent: BaseWall
   id: WallAbductor
-  name: shuttle wall
+  name: nanolaminate wall # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)